tox.ini

```diff
@@ -34,8 +34,9 @@ commands =
     python -c 'import mlx.warnings;print(mlx.warnings.__version__)'
     python -m mlx.warnings -h
     python -m mlx.warnings --version
-    python -m mlx.warnings -j tests/junit*.xml --maxwarnings 3 --minwarnings 3
-    python -m mlx.warnings -j "tests/junit*.xml" --maxwarnings 3 --minwarnings 3 #emulate for windows (no shell expansion)
+    python -m mlx.warnings -j --maxwarnings 3 --minwarnings 3 tests/junit*.xml
+    python -m mlx.warnings -j --maxwarnings 3 --minwarnings 3 "tests/junit*.xml"  #emulate for windows (no shell expansion)
+    python -m mlx.warnings -j --command --maxwarnings 2 --minwarnings 2 cat tests/junit_double_fail.xml
```

Comment:
:+1: ￼ cool test case, do we need quotes around the given command?

Reply:
No, it just needs to be after all arguments otherwise it is considered as command arguments.